ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/poe.j2

```diff
@@ -0,0 +1,65 @@
+{# doc - poe #}
+{% set POE_CLASS_MAP = {0: "15.40", 1: "4.00", 2: "7.00", 3: "15.40", 4: "30.00", 5: "45.00", 6: "60.00", 7: "75.00", 8: "90.00"} %}
+{% set poe_global_defaults = namespace(reboot_action="power-off", interface_shutdown_action="maintain", lldp_negotiate=true) %}
+{% set ethernet_interfaces_poe = [] %}
+{% for ethernet_interface in ethernet_interfaces | arista.avd.natural_sort('name') %}
+{%     if ethernet_interface.poe is arista.avd.defined %}
+{%         do ethernet_interfaces_poe.append(ethernet_interface) %}
+{%     endif %}
+{% endfor %}
+{% if (poe is arista.avd.defined) or (ethernet_interfaces_poe | length > 0) %}
+## Power Over Ethernet (PoE)
+
+### PoE Summary
```

Comment:
Looking at the documentation again we should add a  `### PoE Configuration` as for LLDP - so essentially rendering the global poe template (probably not worth rendering the interface level ones and so the category could be PoE Global Configuration rendered only if `poe` is defined)

```
    ### LLDP Device Configuration

    ```eos
    {%     include 'eos/lldp.j2' %}
    ```
```